src/test/java/com/epam/tc/hw4/steps/ActionStep.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.tc.hw4.steps;
+
+import com.epam.tc.hw4.AbstractChromeTest;
+import io.qameta.allure.Step;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class ActionStep extends AbstractChromeTest {
```

Comment:
Please, create an Abstract or Base Step class, that would be extended by ActionStep and AssertStep classes. You don't have to inherit step class from base test class.